coprocessor/fhevm-engine/host-listener/build.rs

```diff
@@ -7,7 +7,13 @@ use semver::Version;
 use std::{env, fs, path::Path, process::Command};
 
 fn build_contracts() {
-    println!("cargo:rerun-if-changed=../../../host-contracts/contracts");
+    println!(
+        "cargo:rerun-if-changed=../../../host-contracts/contracts/ACL.sol"
+    );
+    println!(
+        "cargo:rerun-if-changed=../../../host-contracts/contracts/ACLEvents.sol"
+    );
+    println!("cargo:rerun-if-changed=../../../host-contracts/contracts/FHEVMExecutor.sol");
```

Comment:
Should we consider implementing the rust bindings for the host-contracts as well? In such case, do you know if there's an issue ticket already?